src/client/app/components/HeaderButtonsComponent.tsx

```diff
@@ -38,16 +38,13 @@ export default class HeaderButtonsComponent extends React.Component<HeaderButton
 				showUIOptions = true;
 				if (renderLogoutButton) {
 					renderAdminButton = true;
-					renderGroupsButton = true;
 				} else {
 					renderLoginButton = true;
 				}
 				break;
 			case 'groups':
 				renderAdminButton = true;
-				break;
-			case 'admin':
-				renderGroupsButton = true;
+				renderGroupsButton = false;
```

Comment:
Why do you hide the groups button on the admin panel?

Reply:
@johndiiorio the group button is still displayed on the admin panel. The case 'admin' was removed entirely. Because now instead of displaying in certain cases, logic is more about  not displaying in certain cases.  'renderGroupbutton' is set true initially now.